src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.image-link {
+  display: block;
+  height: 40px;
+  padding: 10px 0;
```

Comment:
There is no need for this changes, you've already set align-items to container, so it will align them

Reply:
Use flex here to avoid this extra space at the bottom of link